### PR TITLE
[enhance](mtmv)add version and version time for table

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Add version to record data changes in the table

Scope of impact: 

- Transaction related operations
- drop partition
- replace partition

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java [hunk 1]
```diff
@@ -175,6 +177,11 @@ public enum OlapTableState {
 
     private AutoIncrementGenerator autoIncrementGenerator;
 
+    @SerializedName(value = "visibleVersion")
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java:180]: Move these to Table.java 's `TableAttributes`, so no need to update meta version

File: fe/fe-core/src/main/java/org/apache/doris/transaction/TableCommitInfo.java [hunk 2]
```diff
@@ -68,6 +78,10 @@ public void readFields(DataInput in) throws IOException {
                 idToPartitionCommitInfo.put(partitionCommitInfo.getPartitionId(), partitionCommitInfo);
             }
         }
+        if (Env.getCurrentEnvJournalVersion() >= FeMetaVersion.VERSION_129) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/transaction/TableCommitInfo.java:81]: Change it to Gson this time

**Comment by zddr** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
Please add more details in your PR comment

**Review by wuwenchi — APPROVED** [review]

**Comment by zddr** [comment]
run buildall

**Review by morningman — COMMENTED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**